resources/schema.json

```diff
@@ -199,5 +199,16 @@
       "type": "string"
     }
   },
+  "options": {
```

Comment:
@migingreen I just noticed that the `options` needs to go one level deeper. Like with the other properties (title, subtitle ect.).

Reply:
My bad, thanks!